libpod/runtime.go

```diff
@@ -125,6 +125,9 @@ type RuntimeConfig struct {
 	// CNIPluginDir sets a number of directories where the CNI network
 	// plugins can be located
 	CNIPluginDir []string `toml:"cni_plugin_dir"`
+
+	// File to contain the pid of conmon
+	ConmonPidFile string
```

Comment:
Should do this per-container, not per-runtime - we can spawn multiple containers with the same runtime, though podman presently does not. Might want to wait to do this until #548 merges, though, as that will make changes to the container much easier.